airflow/models.py

```diff
@@ -71,7 +71,11 @@
 from airflow.utils.timeout import timeout
 from airflow.utils.trigger_rule import TriggerRule
 
-Base = declarative_base()
+SQL_ALCHEMY_SCHEMA = configuration.get('core', 'SQL_ALCHEMY_SCHEMA')
+if SQL_ALCHEMY_SCHEMA is None or SQL_ALCHEMY_SCHEMA.isspace():
```

Comment:
If not, I would suggest `not SQL_ALCHEMY_SCHEMA.strip()` instead of `SQL_ALCHEMY_SCHEMA.isspace()` because the latter will not catch the case where the value is an empty string.